tests/page-objects/admin/user.wdio.page.js

```diff
@@ -119,6 +136,35 @@ const getContactErrorText = async () => {
   return await (await contactErrorMessage()).getText();
 };
 
+const getSuccessfulyUploadedUsers = async () => {
+  return await (await successfulyUploadedUsers()).getText();
+};
+
+const getPreviouslyUploadedUsers = async () => {
+  return await (await previouslyUploadedUsers()).getText();
+};
+
+const getFailedUploadedUsers = async () => {
+  return await (await failedUploadedUsers()).getText();
+};
+
+const backToUserList = async () => {
+  await (await backToUserListButton()).waitForDisplayed();
+  await (await backToUserListButton()).click();
+};
+
+const openUploadUsersDialog = async () => {
+  await (await uploadUsersButton()).waitForDisplayed();
+  await (await uploadUsersButton()).click();
+  await (await uploadUsersDialog()).waitForDisplayed();
+  // wait for animations to finish
+  await browser.pause(500);
+};
+
+const uploadSummary = async () => {
```

Comment:
I'd think this function uploads summary, I think this name is clearer:
```suggestion
const waitForUploadSummary = async () => {
```